docs/source/quickstart-gcp.rst

```diff
@@ -91,6 +91,7 @@ Start by copying the configuration file shown below.  Using an editor, write thi
 
     [cluster]
     num-nodes = 2
+    labels = owner=${USER}
```

Comment:
For consistency with the rest of the document. In the GCP quickstart, `$USER` is used throughout: https://github.com/ncbi/elastic-blast-docs/blob/master/docs/source/quickstart-gcp.rst#set-up-an-output-bucket-if-one-doesnt-exist

Reply:
I see `$USER` used in the command line for setting up the bucket and I see this as a problem. In the command lines the text "$USER" can be left as it is, because shell will replace it with the username. `labels = owner=${USER}` creates a false impression that `${USER}` in the config file will also be automatically replaced by some appropriate value.

Reply:
> labels = owner=${USER} creates a false impression that ${USER} in the config file will also be automatically replaced by some appropriate value.

That is a reasonable interpretation and to counter that the text below the config file specifies that those configuration entries (i.e.: `results` and `labels`) should be updated. Do you think something else should be done to clarify?

Reply:
OK.